example/ExampleMain.cpp

```diff
@@ -91,7 +91,7 @@ void createFunctionExample(Module &module, const Twine &name) {
   Value *x2 = b.create<xd::Add32Op>(x1, sizeOf32, 11);
   Value *x3 = b.create<xd::CombineOp>(x2, x1);
   Value *x4 = b.create<xd::IExtOp>(b.getInt64Ty(), x3);
-  b.create<xd::WriteOp>(x4);
+  [[maybe_unused]] const xd::WriteOp *x5 = b.create<xd::WriteOp>(x4);
```

Comment:
The change here does not do anything?

Reply:
I forgot to change it. Will revert that change later and then merge the PR.